src/main/java/org/openstreetmap/atlas/geography/atlas/items/complex/bignode/BigNodeFinder.java

```diff
@@ -472,7 +472,7 @@ private boolean isDualCarriageWayJunctionEdge(final Edge candidateEdge)
                 }
                 catch (final CoreException e)
                 {
-                    logger.warn("Could not append dual carriageway route {} with with {}",
+                    throw new CoreException("Could not append dual carriageway route {} with {}",
```

Comment:
This is not an issue, since if that exception was caught, there would have been a subsequent NPE a few lines below.